config-service-impl/src/main/java/org/hypertrace/config/service/store/ConfigStore.java

```diff
@@ -42,6 +43,15 @@ UpsertedConfig writeConfig(
    */
   ContextSpecificConfig getConfig(ConfigResourceContext configResourceContext) throws IOException;
 
+  /**
+   * Get the configs with the latest version for the specified resources.
+   *
+   * @param configResourceContexts
+   * @return
+   */
+  List<ContextSpecificConfig> getConfigs(Set<ConfigResourceContext> configResourceContexts)
```

Comment:
```suggestion
  List<ContextSpecificConfig> getAllConfigs(Set<ConfigResourceContext> configResourceContexts)
```

Reply:
got it. I was confused about the implementation of `getAllConfigs`. We can think about calling `getAllContextConfigs`, but the current naming is fine too